src/sidebar/cross-origin-rpc.js

```diff
@@ -15,13 +28,15 @@
  * http://www.jsonrpc.org/specification
  *
  * The only part that we support so far is receiving JSON-RPC 2.0 requests (not
- * notifications) without any parameters and sending back a successful
- * response. Notifications (JSON-RPC calls that don't require a response),
- * method parameters, and error responses are not yet supported.
+ * notifications) and sending back a successful "ok" response.
  *
+ * All methods called upon must be mapped in the `registeredMethods` function.
  */
 // @ngInject
 function start(store, settings, $window) {
+  // get the mapped methods.
```

Comment:
I think this comment is not necessary. If you're worried about clarity, you could rename the reference from `methods` to `allowedMethods` or something.